### ISSUE TITLE
How to add tooltip for appended or prepended icons in vuetify-form-base ?

### ISSUE THREAD

**Issue by dileep524** [wotamann/vuetify-form-base#29]
```
<template>
  <v-container fluid>
    <h4>Textfields and Fileselector with Mask and Tooltips</h4>

    <v-form>    
      <v-form-base
        :value="myValue"
        :schema="mySchema"
        @input="log"
      />

    </v-form>

    <infoline
      :value="myValue"
      :schema="mySchema"
      :path="$options._componentTag"
    />
  </v-container>
</template>

<script>
import VFormBase from '@/components/vFormBase'
import Infoline from '@/components/infoline'
import log from '@/lib'

const mask = '####-####-####-####'

const accept = 'image/*'

export default {
  name: 'Textfields',
  components: { VFormBase, Infoline },
  data () {
    return {
      myValue: {
        name: 'Base',
        creditcard: '12345678',
        password: 'abcdefgh',
        file: [] // array of File objects
      },      
      mySchema: {        
        name: 'text',         
        password: { 
          type: 'password', 
          clearable: true, 
          flex: 12 
        },
        creditcard: { 
          type: 'text', 
          label: 'Creditcard', 
          prependInnerIcon: 'credit_card', 
          hint: mask, 
          mask, 
          tooltip: 'Creditcard', 
          flex: 12 
        },
        file: { 
          type: 'file', 
          label: 'Files', 
          accept, 
          multiple: true, 
          tooltip: { color: 'green', label: 'File Selection', top: true }, 
          flex: 12 
        }
      }
    }
  },
  methods: {
    log
  }
}
</script>
```
From the above code tooltip is applied for entire textfield by using tooltip: 'Creditcard' but i want tooltip only for icon. Now tooltip is applied on hover of entire text field but it should display only on hover of icon.

Reference :1. https://github.com/wotamann/vuetify-form-base 2. https://wotamann.github.io/

**Comment by wotamann** [comment]
I have been looking for but apparently vuetify 2.0 doesn't support binding v-tooltip to prepend-inner-icon

```
  //   does not work !!!!
 <v-tooltip left slot="prepend-inner">
        <template v-slot:activator="{ on }" >        
               <v-text-field v-on="on" prepend-inner-icon="add" >                      
               </v-text-field>
        </template>      
        <span>Left tooltip</span>     
 </v-tooltip>
```
tl;dr This means that without adapting **vuetify-form-base**  directly, it is not feasible.  

_Originally posted by @wotamann in https://github.com/wotamann/vuetify-form-base/issues/26#issuecomment-581559228_

----

Hello, it works if the tooltip does not act as a wrapper of the field. Had solved it with an SFC at first, but I thought it completely twisted the logic of your work. Works also with "prepend-inner".
```

<v-text-field> 
     <v-tooltip slot="append-outer"> 
         <template v-slot:activator="{on}"> 
             <v-icon @... v-on="on"> Tooltip icon </v-icon> 
         </template> <span>Tooltip content</span> 
     </v-tooltip> 
 </v-text-field>
```

_Originally posted by @Rambone in https://github.com/wotamann/vuetify-form-base/issues/26#issuecomment-581625967_

**Comment by dileep524** [comment]
But how can we get it from vuetify schema ?

**Comment by dileep524** [comment]
can we have icon component seperately

"icon": { "type": "icon", "title": "add"}, something like this

**Comment by wotamann** [comment]
Try this, it will display 2x the Print-Icon
```
myValue: {
      "iconV": "print",
   }
},
mySchema: {
     iconV: {
         type: "icon"
     },
     iconL: {
         "type": "icon",
         "label": "print",
    }
}
```
